src/types/GradeLevel.ts

```diff
@@ -46,6 +46,14 @@ namespace GradeLevel {
   export function toString(level: GradeLevel): string {
     return toStringMappings[level];
   }
+
+  export function includesGrade(
+    grades?: number[],
+    paramGrades?: number[]
+  ): boolean {
+    if (!paramGrades || !grades) return true;
+    return grades.some((g) => paramGrades.includes(g));
```

Comment:
This can be written as one line:
```suggestion
    return !paramGrades || !grades || grades.some((g) => paramGrades.includes(g));
```

Actually, other filters will probably be similar (e.g. majors, states, etc.) so I suggest either inlining this or refactoring this as a generic function in Scholarships.ts

Reply:
I refactored this as a generic function